### DISCUSSION TITLE
Some warnings with PdfReader

### DISCUSSION THREAD

**Discussion by Vadkoz** [py-pdf/pypdf#1448] [Q&A] [answered]
I need to get a count of the pages of the pdf file.
Got the next warnings when reading pdf via PdfReader or PdfFileReader: 
Multiple definitions in dictionary at byte **** for key /Info
incorrect startxref pointer(*some number (i got 1 and 3)*)

But I got a count of pages anyway, despite any warnings. I need to know if the count I got is correct. Maybe that warnings means something important that may interfere with the receipt of information, I don't know. So I checked several files, but I'm still not sure about it.
Thanks!

**Comment by MartinThoma** [comment, marked as answer]
Please see https://pypdf2.readthedocs.io/en/latest/user/robustness.html and https://pypdf2.readthedocs.io/en/latest/user/suppress-warnings.html
